lms/envs/common.py

```diff
@@ -446,8 +446,16 @@
         'write': 'Write scope',
         'email': 'Email scope',
         'profile': 'Profile scope',
-    }
+        # The identity scope grants a basic permission to get
+        # the user's preferred_username. This can be used by
+        # 3rd party apps to link accounts. No other data is exchanged if
+        # this is the only scope requested
+        'identity': 'Receive basic identity information, such as your public username',
```

Comment:
The profile scope will provide the username and real name. See https://github.com/edx/edx-oauth2-provider/blob/master/edx_oauth2_provider/oidc/handlers.py#L140-L170.